genrl/deep/agents/base.py

```diff
@@ -54,26 +52,26 @@ def select_action(self, state: np.ndarray) -> np.ndarray:
         :returns: Action based on the state and epsilon value
         :rtype: int, float, ...
         """
-        raise NotImplementedError()
-
-    def get_loss(self) -> None:
-        """
-        Calculate and return the total loss
-        """
-        raise NotImplementedError()
```

Comment:
Why is this removed?

Reply:
The `OnPolicyAgent` uses `get_traj_loss` and the `OffPolicyAgent` will most likely use `get_q_loss` and `get_p_loss`. It's mostly that the name get_loss might be too general. I removed everything that's not being used right away. I'll add the functions back when needed.